frontend/src/components/LandingPage/Components/Footer/Footer.js

```diff
@@ -17,11 +17,11 @@ const Footer = () => {
 
 
                     <div className="social">
-                        <a href="https://github.com/dvstechlabs/Noteslify"><i className="fa-brands fa-github-square"></i></a>
-                        <a href="#test"><i className="fa-brands fa-youtube-square"></i></a>
-                        <a href="#test"><i className="fa-brands fa-facebook-square"></i></a>
-                        <a href="#test"><i className="fa-brands fa-twitter-square"></i></a>
-                        <a href="#test"><i className="fa-brands fa-youtube-square"></i></a>
+                        <a title="github" href="https://github.com/dvstechlabs/Noteslify"><i className="fa-brands fa-github-square"></i></a>
```

Comment:
```suggestion
                        <a title="GitHub" href="https://github.com/dvstechlabs/Noteslify"><i className="fa-brands fa-github-square"></i></a>
```
Original spelling should be there 🤔